test/e2e/test/elasticsearch/license_fixture.go

```diff
@@ -58,12 +58,12 @@ func GenerateTestLicense(signer *license.Signer) (client.License, error) {
 	licenseSpec := client.License{
 		UID:                uuid.String(),
 		Type:               "enterprise",
-		IssueDateInMillis:  chrono.ToMillis(time.Now()),
+		IssueDateInMillis:  chrono.ToMillis(time.Now().Add(-1 * time.Hour)),
```

Comment:
Should we take this a step further and set the issue date to eg. 3 days earlier (which also seems to be a realistic thing we'd do when generating the license)? So we avoid any potential timezone inconsistency.